modules/raft/src/main/java/org/apache/ignite/raft/jraft/storage/DestroyStorageIntentStorage.java

```diff
@@ -0,0 +1,15 @@
+package org.apache.ignite.raft.jraft.storage;
+
+import java.util.Collection;
+
+/** Persists and retrieves intent to complete storages destruction on node start. */
+public interface DestroyStorageIntentStorage extends Storage {
```

Comment:
```suggestion
public interface GroupStoragesDestructionIntents extends Storage {
```
Also:

1. I don't think it should reside in jraft package
2. I double it has to extend `Storage`